### PR TITLE
RHDEVDOCS-4441 - Logging 5.3.14 Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.9+

Issue: https://issues.redhat.com/browse/RHDEVDOCS-4441

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cluster-logging-rn-5.3.14.adoc [hunk 1]
```diff
@@ -0,0 +1,84 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-3-14{context}"]
```
Comment (gabriel-rh) [inline modules/cluster-logging-rn-5.3.14.adoc:3]: Usually there is an _ before {context} but I don't think it really matters here. Only change it if you feel like it.

**Review by gabriel-rh — APPROVED** [review]
/lgtm

File: modules/cluster-logging-rn-5.3.14.adoc [hunk 2]
```diff
@@ -0,0 +1,84 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-3-14_{context}"]
+= Logging 5.3.14
+This release includes link:https://access.redhat.com/errata/RHSA-2022:8889[OpenShift Logging Bug Fix Release 5.3.14].
+
+[id="openshift-logging-5-3-14-bug-fixes"]
+== Bug fixes
+None.
+
```
Comment (kabirbhartiRH) [inline modules/cluster-logging-rn-5.3.14.adoc:10]: @libander Can you add [LOG-3293](https://issues.redhat.com/browse/LOG-3293)

**Review by anpingli — COMMENTED** [review]
/lgtm

File: modules/cluster-logging-rn-5.3.14.adoc [hunk 3]
```diff
@@ -0,0 +1,84 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-3-14_{context}"]
+= Logging 5.3.14
+This release includes link:https://access.redhat.com/errata/RHSA-2022:8889[OpenShift Logging Bug Fix Release 5.3.14].
+
+[id="openshift-logging-5-3-14-bug-fixes"]
+== Bug fixes
+* Before this update, the log file size map generated by the `log-file-metrics-exporter` component did not remove entries for deleted files, resulting in an increase file size, and process memory. With this update, the log file size map does not contain entries for deleted files. (link:https://issues.redhat.com/browse/LOG-3293[LOG-3293])
```
Comment (bburt-rh) [inline modules/cluster-logging-rn-5.3.14.adoc:9]:
```suggestion
* Before this update, the log file size map generated by the `log-file-metrics-exporter` component did not remove entries for deleted files, resulting in increased file size and process memory. With this update, the log file size map does not contain entries for deleted files. (link:https://issues.redhat.com/browse/LOG-3293[LOG-3293])
```

**Pull request merged**